src/utils.js

```diff
@@ -24,4 +29,73 @@ module.exports = {
       []
     );
   },
+
+  // Synchronously walk a directory of compiled assets, read each file and calculate its SHA 256
+  // hash, and create an array of Resource objects.
+  gatherBuildResources(percyClient, rootDir, options = {}) {
+    // The base of the URL that will be prepended to every resource URL, such as "/assets".
+    options.baseUrlPath = options.baseUrlPath || '';
+    options.skippedPathRegexes = options.skippedPathRegexes || [];
```

Comment:
Slightly refactored things this method didn't have before: extracting and exposing a defined API for skipping resources, passing in a baseUrlPath, and configuring followLinks.